app/assets/stylesheets/modules/spotlight_overrides.scss

```diff
@@ -157,7 +157,19 @@ h5,
 }
 
 .blacklight-about_pages-show #sidebar {
```

Comment:
@ggeisler the ticket specifies that this should only affect the about page.  Just want to validate that.

Reply:
@jkeck Yes, definitely. Just trying to tighten up the very loose spacing in the About page sidebar.